src/content/docs/tutorial-service-level-mgmt/improve-with-slm.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: "Improve with service level management"
+metaDescription: "New Relic observability maturity series: our service level management guide shows you how to easily measure and communicate the overall health, performance, and quality of your digital products and services to all stakeholders."
+redirects:
+freshnessValidatedDate: never
+---
+
+After creating your SLIs, you can use them to continously improve your system via multiple different processes. Below, you'll find different ways to use your SLIs across many different areas of the New Relic platform. 
```

Comment:
```suggestion
After creating your SLIs, you can use them to continuously improve your system via multiple different processes. Below, you'll find different ways to use your SLIs across many different areas of the New Relic platform. 
```

Reply:
I found a more simple word to use, but this is fixed either way!